website/src/pages/documents/document.css.ts

```diff
@@ -182,6 +182,7 @@ export const contentSection2 = style([
   sprinkles({ display: { any: "none", medium: "block" } }),
   {
     flex: 1,
+    maxWidth: "33vw",
```

Comment:
Using `vw` units doesn't really make sense for desktop (this is for desktop right?), because the content is at a fixed width. So this would make more sense as a `px` value or `rem` value perhaps.